gelman_rubin.py

```diff
@@ -1,44 +1,73 @@
-<<<<<<< HEAD
-gelman rubin..
-=======
-
 import numpy as np
+"""This is the implementation of the revised Gelman-Rubin Diagnostic
+with use of the presented calculations given in the paper
+https://arxiv.org/pdf/1812.09384.pdf
+written by Dootika Vats and Christina Knudson
+"""
```

Comment:
```suggestion
"""This is the implementation of the revised Gelman-Rubin Diagnostic
with use of the presented calculations given in the paper
https://arxiv.org/pdf/1812.09384.pdf
written by Dootika Vats and Christina Knudson
"""
import numpy as np
```
Module doc strings always go to the very top of the file, before `import` statements.